subprojects/packagefiles/cpp-semver/meson.build

```diff
@@ -0,0 +1,24 @@
+project(
+  'cpp-semver',
+  'cpp',
+  version: '0.3.3',
+  license: 'MIT',
+  meson_version: '>=1.2.0',
+  default_options: {
+    'cpp_std': ['c++17'],
+  },
+)
+
+install_subdir(
+  'include',
+  install_dir: get_option('includedir'),
+)
+
+cpp_semver_dep = declare_dependency(
+  include_directories: include_directories('include'),
+  version: meson.project_version(),
+)
```

Comment:
```suggestion
meson.override_dependency('cpp-semver', cpp_semver_dep)
```

Reply:
I always thought, that was bad practice to do that, but now I know otherwise, thnaks 👍🏼